openshift/catalogd/manifests/22-servicemonitor-catalogd.yml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: catalogd-metrics-monitor
+  namespace: openshift-catalogd
+  labels:
+    app.kubernetes.io/name: catalogd
+    app.kubernetes.io/part-of: olm
+    openshift.io/cluster-monitoring: 'true'
+spec:
+  endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
+      path: /metrics
+      port: metrics
+      scheme: https
+      tlsConfig:
+        caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+        certFile: /etc/prometheus/secrets/metrics-client-certs/tls.crt
+        keyFile: /etc/prometheus/secrets/metrics-client-certs/tls.key
```

Comment:
My gut tells me these aren't necessary... I could be wrong.

Reply:
Because all inside the cluster (Prometheus and OLM compontets) probably do not need to pass the certs.
That is good point. It might work with RBAC permissions only.

Reply:
These would be the client certificates that prometheus presents to our endpoint when it scrapes it. I don't think we have configured client-cert-based auth. I would think the service account token specified in line 13 would be what we use to authenticate.

Reply:
Ye, that's why I think they're unnecessary, they'd be used for client authentication, which we don't have turned on, and very likely would require adding _another_ CA...